spring-cloud-stream-samples/rxjava/src/main/java/demo/RxJavaApplication.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package demo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+import rxjava.RxJavaTransformer;
+
+/**
+ * @author Ilayaperumal Gopinathan
+ */
+@SpringBootApplication
+@ComponentScan(basePackageClasses= RxJavaTransformer.class)
```

Comment:
I followed the existing samples template. Here is the PR that cleans up things related to this: https://github.com/spring-cloud/spring-cloud-stream/pull/118